src/components/Toast/useToast.test.tsx

```diff
@@ -22,18 +22,33 @@ describe("useToast", () => {
     // Then expect it to be closed
     expect(r.query.toast).not.toBeInTheDocument();
   });
+
+  it("can clear a toast", async () => {
+    // Given an app that can trigger a toast
+    const r = await render(<TestToastComponent />);
+    // When triggering the toast and clicking on the clear button
+    click(r.toastButton);
+    click(r.clearToast);
+    // Then expect it to be cleared
+    expect(r.query.toast).not.toBeInTheDocument();
+  });
 });
 
 function TestToastComponent() {
-  const { showToast } = useToast();
+  const { showToast, clear } = useToast();
   const showToastOnClick = () => {
     showToast({ message: "Test Toast", type: "error" });
   };
 
+  const clearToast = () => {
+    clear();
+  };
```

Comment:
nit: this wrapping function isn't doing anything - maybe just pass `clear` directly to the `onClick`?